test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -0,0 +1,311 @@
+#ifndef PSQLODBC_TESTS_COMMON_H
+#define PSQLODBC_TESTS_COMMON_H
+
+#include <cmath>
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include <string>
+#include <time.h>
+#include <vector>
+
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+
+using std::vector;
+using std::string;
+using std::pair;
+
+const int BUFFER_LENGTH = 8192;
+const int BUFFER_SIZE = 256;
+const int CHARSIZE = 255;
+const int INT_BYTES_EXPECTED = 4;
+
+string InitializeInsertString(const vector<string>& insertedValues, bool isNumericInsert);
```

Comment:
Comments on what each of the functions please.
Having an example would be an extra nice to have.
ie for table_name, is it just the object name or does it also include the database and schema

Reply:
I am working on this and will add these comments in the next commit. I pushed the commit addressing your other comments related to functional code changes.

Reply:
Added docstrings. Done.